_includes/scenarios/integrate/request-and-response-introduction.md

```diff
@@ -0,0 +1,49 @@
+Also, Responses are unique.
+Once a Request is answered, it cannot be answered again, e.g. on another device, if the App is installed on multiple ones, having all received the Request.
+This ensures an unambiguous functioning.
+
+## Types of Request- and ResponseItems
+
+The core of a [Request]({% link _docs_integrate/data-model-overview.md %}#request) is the [RequestItem]({% link _docs_integrate/data-model-overview.md %}#requestitem).
+It specifies what you want from the other Identity.
+There are many different types all described in detail in the [Requests and RequestItems]({% link _docs_integrate/requests-and-requestitems.md %}) section.
+Here, we just want to give a brief overview:
+
+- [AuthenticationRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#authenticationrequestitem)s can be used to request an authentication, e.g. for a login to a website
+- [ConsentRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#consentrequestitem)s can be used to request a consent to an freely configurable text, e.g. to receive a newsletter
+- [CreateAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#createattributerequestitem)s can be used to create an Identity- or RelationshipAttribute with a fixed value for a peer, e.g. a certificate
+- [FreeTextRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#freetextrequestitem)s can be used to send a free text to a contact, that in turn can accept this with a free text as well
+- [ProposeAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#proposeattributerequestitem)s can be used to ask the peer for Attributes, already suggesting answers, e.g. asking for an address and setting the country of the organization as default
+- [ReadAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem)s can be used to ask a peer for Attributes, e.g. asking for an address
+- [ShareAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#shareattributerequestitem)s can be used to share own IdentityAttributes with a contact, e.g. sharing the own DisplayName when requesting a new Relationship
+
+In case multiple RequestItems should be answered jointly, e.g. to enhance the structure for the user, you can combine them to a [RequestItemGroup]({% link _docs_integrate/data-model-overview.md %}#requestitemgroup).
+RequestItems or RequestItemGroups are then embedded in a Request, which can be sent to the other Identity.
+This happens either via a Template (see [Requests over Templates]({% link _docs_integrate/requests-over-templates.md %})) or via a Message (see [Requests over Messages]({% link _docs_integrate/requests-over-messages.md %})).
+
+Once the other Identity receives a Request, they can decide whether they want to accept or reject the corresponding RequestItems or RequestItemGroups.
```

Comment:
I'm missing following topics in the file:
- title/description of RequestItems
- mustBeAccepted property and how it works together
- proprietary metadata of RequestItems

Reply:
Correct me if I'm wrong, but the RequestItems page is more or less a description of data structures like a reference. The questions "why is this property there" and "how can I use the property correctly" are not answered. We shouldn't start explaining that mustBeAccepted is a boolean prop, etc. But at least touch the topic that there are optional/required RequestItems inside a Request and that the mustBeAccepted property should be used for it.

AFAIK, we have the whole overarching datamodel overview: https://enmeshed.eu/integrate/data-model-overview#requestitem and the list of each specific request item: https://enmeshed.eu/integrate/requests-and-requestitems

My gut feeling is, that this page here should act as an introduction to the whole topic of Requests, explain the generic structure of Requests, RequestItemGroups and RequestItems in a nutshell, and link to the specific documentations as a reference.